src/panel/poi/ActionButtons.jsx

```diff
@@ -42,9 +47,17 @@ const ActionButtons = ({
 
     <Button
       className="poi_panel__action__favorite"
-      onClick={toggleStorePoi}
-      icon={isPoiInFavorite ? 'icon_star-filled' : 'star'}
+      data-active={isPoiInFavorite}
       title={_('Favorites', 'poi panel')}
+      onClick={toggleStorePoi}
+      style={{ borderColor: favoriteColor }}
+      icon={
+        <Heart
+          width={16}
```

Comment:
It would be nice to find a way to omit this explicit size, but let's move on and think about this later :+1: